### ISSUE TITLE
RxJs version for update method doesn't will work with several observables.

### ISSUE THREAD

**Issue by xgrommx** [milankinen/ffux#2]
Hi @milankinen. Your implementation https://github.com/milankinen/ffux/blob/master/lib/ffux-rx.js#L37 doesn't will work with several observables like in Bacon. I have created another implementation early. My implementation will work with array of obsevables like so:

``` js
Rx.Obsevable.update(0, [obs1, obs2], (prev, o1, o2) => prev + o1 + o2).subscribe(::console.log);
```

You can look on this https://github.com/xgrommx/react-rx-flux/blob/master/src/store.js#L7

**Comment by milankinen** [comment]
Very nice, thanks! Will be fixed in the next release

**Comment by milankinen** [comment]
Fixed in `0.7.0`
